### PR TITLE
tool: update the tool downloading links and instructions

### PR DESCRIPTION
**Pull request by qiancai**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.1 (TiDB 6.1 versions)
- [x] v6.0 (TiDB 6.0 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: dm/dm-faq.md [hunk 1]
```diff
@@ -213,7 +213,7 @@ if the DDL is not needed, you can use a filter rule with \"*\" schema-pattern to
 - 方法一：使用 `tiup update --self && tiup update dm` 升级 TiUP 到更新版本，随后先缩容再扩容集群中的 grafana 节点，重建 grafana 服务。
 - 方法二：
     1. 备份 `deploy/grafana-$port/bin/public` 文件夹。
-    2. 下载 [TiUP DM 离线镜像包](https://download.pingcap.org/tidb-dm-v2.0.1-linux-amd64.tar.gz)，并进行解压，将其中的 grafana-v4.0.3-**.tar.gz 文件解压后，用解压出的 public/ 文件夹替换前面所描述的文件夹，运行 `tiup dm restart $cluster_name -R grafana` 重启 grafana 服务监控。
+    2. 下载 [TiUP DM 离线镜像包](https://download.pingcap.org/tidb-dm-v2.0.1-linux-amd64.tar.gz)，并进行解压，将其中的 grafana-{version}-**.tar.gz 文件解压后，用解压出的 public/ 文件夹替换前面所描述的文件夹，运行 `tiup dm restart $cluster_name -R grafana` 重启 grafana 服务监控。
```
Comment (qiancai) [inline dm/dm-faq.md:216]:
```suggestion
    2. 下载 [TiUP DM 离线镜像包](https://download.pingcap.org/tidb-dm-v2.0.1-linux-amd64.tar.gz)，并进行解压，将其中的 grafana-v4.0.3-**.tar.gz 文件解压后，用解压出的 public/ 文件夹替换前面所描述的文件夹，运行 `tiup dm restart $cluster_name -R grafana` 重启 grafana 服务监控。
```

**Review by lichunzhu — APPROVED** [review]
LGTM

**Review by sunzhaoyang — APPROVED** [review]
LGTM

**Review by qqqdan — APPROVED** [review]

File: br/backup-and-restore-tool.md [hunk 2]
```diff
@@ -155,7 +155,7 @@ TiDB 支持使用 SQL 语句 [`BACKUP`](/sql-statements/sql-statement-backup.md#
 
 #### 通过命令行工具
 
-TiDB 支持使用 BR 命令行工具进行备份恢复（需[手动下载](/download-ecosystem-tools.md#备份和恢复-br-工具)）。关于 BR 命令行工具的具体使用方法，请参阅[使用备份与恢复工具 BR](/br/use-br-command-line-tool.md)。
+TiDB 支持使用 BR 命令行工具进行备份恢复（需[手动下载](/download-ecosystem-tools.md)）。关于 BR 命令行工具的具体使用方法，请参阅[使用备份与恢复工具 BR](/br/use-br-command-line-tool.md)。
```
Comment (leoppro) [inline br/backup-and-restore-tool.md:158]: BR 已经不需要手动下载了 tiup 有支持

**Review by nolouch — COMMENTED** [review]
PD part LGTM.

**Review by leoppro — APPROVED** [review]
LGTM

File: br/backup-and-restore-tool.md (hunk 2 above)
Comment (qiancai) [reply br/backup-and-restore-tool.md:158]: 是的，这个 PR 先只修复这里的文档链接更新问题。BR 工具使用相关内容，目前已经在 https://github.com/pingcap/docs-cn/pull/8273/files (还在 review 中） 中优化了。

File: dm/quick-start-create-task.md [hunk 3]
```diff
@@ -69,7 +69,7 @@ docker run --rm --name mysql-3307 -p 3307:3307 -e MYSQL_ALLOW_EMPTY_PASSWORD=tru
 {{< copyable "shell-regular" >}}
 
 ```bash
-wget https://download.pingcap.org/tidb-latest-linux-amd64.tar.gz
+wget https://download.pingcap.org/tidb-community-server-v6.0.0-linux-amd64.tar.gz
```
[the remark this answers is not in the document]
Comment (qiancai) [reply dm/quick-start-create-task.md:72]: According to our release manager, exposing the nightly image download link is not allowed.

**Review by shichun-0415 — APPROVED** [review]

File: download-ecosystem-tools.md [hunk 4]
```diff
@@ -5,85 +5,44 @@ aliases: ['/docs-cn/dev/download-ecosystem-tools/','/docs-cn/dev/reference/tools
...
-## 备份和恢复 (BR) 工具
-
-使用下表中的链接下载 [BR 工具](/br/backup-and-restore-tool.md)：
-
-| 安装包 | 操作系统 | 架构 | SHA256 校验和 |
-|:---|:---|:---|:---|
-| `https://download.pingcap.org/tidb-toolkit-{version}-linux-amd64.tar.gz` | Linux | amd64 | `https://download.pingcap.org/tidb-toolkit-{version}-linux-amd64.sha256` |
+TiDB 工具包中包含了一些常用的 TiDB 工具，例如数据导出工具 Dumpling、数据导入工具 TiDB Lightning、备份恢复工具 BR。
 
 > **注意：**
```
Comment (ran-huang) [inline download-ecosystem-tools.md:12]:
```suggestion
> **建议：**
```

File: download-ecosystem-tools.md [hunk 5]
```diff
@@ -5,85 +5,44 @@ aliases: ['/docs-cn/dev/download-ecosystem-tools/','/docs-cn/dev/reference/tools
...
-
-| 安装包 | 操作系统 | 架构 | SHA256 校验和 |
-|:---|:---|:---|:---|
-| [tidb-enterprise-tools-nightly-linux-amd64.tar.gz](https://download.pingcap.org/tidb-enterprise-tools-nightly-linux-amd64.tar.gz) | Linux | amd64 | [tidb-enterprise-tools-nightly-linux-amd64.sha256](https://download.pingcap.org/tidb-enterprise-tools-nightly-linux-amd64.sha256) |
-
-## TiCDC
-
-[TiCDC](/ticdc/ticdc-overview.md) 的下载方式见 [TiCDC 安装部署文档](/ticdc/deploy-ticdc.md)。
+> - 点击**立即下载**后，默认下载当前 TiDB 的最新发布版本。如需下载历史版本，请在 [TiDB 社区版](https://pingcap.com/zh/product-community/) 页面底部查看历史版本下载信息。
+> - 如果需要在 Kubernetes 上部署运维 TiDB，无需下载 TiDB-community-toolkit 软件包， 请参考[离线安装 TiDB Operator](https://docs.pingcap.com/zh/tidb-in-kubernetes/stable/deploy-tidb-operator#离线安装-tidb-operator)。
```
Comment (ran-huang) [inline download-ecosystem-tools.md:30]:
```suggestion
> - 如果需要在 Kubernetes 上部署运维 TiDB，无需下载 TiDB-community-toolkit 软件包，请参考[离线安装 TiDB Operator](https://docs.pingcap.com/zh/tidb-in-kubernetes/stable/deploy-tidb-operator#离线安装-tidb-operator)。
```

File: tidb-lightning/deploy-tidb-lightning.md [hunk 6]
```diff
@@ -68,7 +68,7 @@ aliases: ['/docs-cn/dev/tidb-lightning/deploy-tidb-lightning/','/docs-cn/dev/ref
 
 #### 第 2 步：下载 TiDB Lightning 安装包
 
-在[工具下载](/download-ecosystem-tools.md#tidb-lightning)页面下载 TiDB Lightning 安装包（TiDB Lightning 完全兼容较低版本的 TiDB 集群，建议选择最新稳定版本）。
+参考 [工具下载](/download-ecosystem-tools.md)文档下载 TiDB Lightning 安装包（TiDB Lightning 完全兼容较低版本的 TiDB 集群，建议选择最新稳定版本）。
```
Comment (ran-huang) [inline tidb-lightning/deploy-tidb-lightning.md:71]:
```suggestion
参考[工具下载](/download-ecosystem-tools.md)文档下载 TiDB Lightning 安装包（TiDB Lightning 完全兼容较低版本的 TiDB 集群，建议选择最新稳定版本）。
```

**Review by ran-huang — APPROVED** [review]
Rest LGTM

File: download-ecosystem-tools.md (hunk 4 above)
Comment (ran-huang) [reply download-ecosystem-tools.md:12]: It'll be rendered like this: 
image

**Comment by qiancai** [comment]
> Rest LGTM

Thanks Ran👍 I've implemented your comments.

**Pull request merged**